R/plot_carp.R

```diff
@@ -82,160 +83,76 @@
 #' }
 plot.CARP <- function(x,
                       ...,
-                      type = c("dendrogram", "path", "dynamic_path", "js", "interactive"),
+                      type = c("dendrogram", "path", "heatmap"),
+                      dynamic = FALSE,
+                      interactive = FALSE,
                       axis = c("PC1", "PC2"),
                       dend.branch.width = 2,
                       dend.labels.cex = .6,
                       dend.ylab.cex = 1.2,
                       percent,
                       k,
-                      percent.seq = seq(0, 1, length.out = 100),
-                      max.nclust = 9,
-                      min.nclust = 1) {
+                      percent.seq = seq(0, 1, 0.01),
+                      slider_y = -0.3) {
 
   type <- match.arg(type)
   switch(
     type,
     dendrogram = {
-      carp_dendro_plot(x,
-                       percent = percent,
-                       k = k,
-                       dend.branch.width = dend.branch.width,
-                       dend.labels.cex = dend.labels.cex,
-                       dend.ylab.cex = dend.ylab.cex,
-                       ...)
+      if (interactive == FALSE) {
+        if (dynamic == FALSE) {
+          carp_dendro_plot(x,
+                           percent = percent,
+                           k = k,
+                           dend.branch.width = dend.branch.width,
+                           dend.labels.cex = dend.labels.cex,
+                           dend.ylab.cex = dend.ylab.cex,
+                           ...)
+        } else {
+          crv_error("Not implemented.")
```

Comment:
Can you be a bit more specific in this error message (saying what isn't implemented.)